wallet/test/Test/Pos/Types/Identity/BinarySpec.hs

```diff
@@ -0,0 +1,19 @@
+-- | This module tests Binary instances.
+
+module Test.Pos.Types.Identity.BinarySpec
+       ( spec
+       ) where
+
+import           Test.Hspec            (Spec, describe)
+import           Universum
+
+import           Pos.Util.BackupPhrase (BackupPhrase)
+
+import           Test.Pos.Helpers      (binaryTest)
+import           Test.Pos.Util         (withDefConfiguration, withDefInfraConfiguration)
```

Comment:
And (apparently last one) another file with wrong import formatting (I am permissive regarding everything in wallet-new, but not in other places).